.github/workflows/run_test_cases.yaml

```diff
@@ -10,26 +10,71 @@ on:
   pull_request:
 
 jobs:
-    run_proper_test:
+    prepare:
         runs-on: ubuntu-20.04
         container: emqx/build-env:erl23.3.4.9-3-ubuntu20.04
+        outputs:
+          ct_apps: ${{ steps.run_find_apps.outputs.ct_apps }}
+        steps:
+          - uses: actions/checkout@v2
```

Comment:
please use latest versions of actions when possible

Reply:
all actions are now set to currently latest